code/modules/medical/genetics/baseBioeffect.dm

```diff
@@ -52,7 +52,8 @@ ABSTRACT_TYPE(/datum/bioEffect)
 	var/req_mut_research = null // If set, need to research the mutation before you can do anything w/ this one
 	var/reclaim_mats = 10 // Materials returned when this gene is reclaimed
 	var/reclaim_fail = 5 // Chance % for a reclamation of this gene to fail
-	var/curable_by_mutadone = 1
+	var/curable_by_mutadone = TRUE //! if 0/FALSE, we cant mutadone this - reinforced, magic genes and anti-toxins use this
+	var/curable_by_vanilla = TRUE //! only for magical genes like clown clumsiness or hairy trait! - we REALLY dont want to lose this
```

Comment:
This var would be better named `is_magical` or similar, since it really just means "never ever remove this (unless)"